compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -515,19 +515,19 @@ unzip3 : [(a, b, c)] -> ([a], [b], [c])
 unzip3 = foldr (\(a,b,c) ~(as,bs,cs) -> (a::as,b::bs,c::cs))
                   ([],[],[])
 
--- | `traceRaw msg a` prints `msg` and returns `a`, for debugging purposes.
+-- | `traceRaw msg a` prints `msg` and returns `a`, for debugging purposes, depending on your loglevel.
```

Comment:
```suggestion
-- | `traceRaw msg a` prints `msg` and returns `a`, for debugging purposes.
--
-- Note that on some ledgers, those messages are not display at the default log level. For Sandbox, you can use `--log-level=debug` to include them.
```
How about that? This makes it clear that this is ledger specific and does not apply to daml script and triggers, it shows an easy way to do it for sandbox while not claiming that this is the only way.

I do agree we need a proper docs section on logging configuration but that’s clearly not for this PR.

Reply:
Sure, that's fine by me.
(You have a typo: s/display/displayed)

Reply:
That’s what I get for looking at PRs before having had coffee :)